internetarchive/bookreader
# ES6 Plugin: Mobile Nav

Plugin now uses:
- `() => {}` where possible without breaking scopes & closures
- string templates for DOM
- tests

### To Test:
- pull down branch
- update your node modules, cmd: `npm install`
- run your local server, cmd: `npm run serve'
- go to page with a browser width lower than 800px: `http://127.0.0.1:8080/BookReaderDemo/demo-ia-olivertwist.html`
- Mobile menu should appear
- Then click Menu 🍔 at top right -> menu slides in from left
- click 'Settings' -> Settings menu slides in
- Click on every Option in Settings -> they all work as expected
- click 'About This Book' -> About menu slides in 
- Book Info is displayed, link in menu changes URL to relative path `~/details/<bookID>`
- click 'Share This Book' -> share menu slides in
- Click on every Option in Share -> the allowed share options pop up their info, note URL is a relative path.


Internal ticket: https://webarchive.jira.com/browse/WEBDEV-3201
